web/packages/core/src/internal/player/inner.tsx

```diff
@@ -2023,7 +2031,9 @@ export class InnerPlayer {
         }
 
         errorArray.push(this.getPanicData());
-
+    
+        this.element.dispatchEvent(new CustomEvent(InnerPlayer.PLAYER_ERROR, error));
```

Comment:
You can pass custom data in `detail`:

```suggestion
        this.element.dispatchEvent(new CustomEvent(InnerPlayer.PANICKED, { detail: { error } }));
```